tool/evaluate-file.ts

```diff
@@ -61,6 +70,16 @@ export function initArgparse(subparsers : argparse.SubParser) {
         choices: ['thingtalk', 'dlgthingtalk'],
         help: `The programming language to generate`
     });
+    parser.add_argument('--tokenized', {
+        action: 'store_true',
+        help: 'The utterances are tokenized.',
+        default: true
+    });
+    parser.add_argument('--untokenized', {
```

Comment:
In other files we call this `--no-tokenized`, and we try to keep consistent using `--no-` for negating boolean flags.